src/main/java/edu/wisc/my/rssToJson/service/RsstoJsonServiceImpl.java

```diff
@@ -51,6 +54,37 @@ public JSONObject getJsonFromURL(String endpoint) {
         }
         jsonToReturn.put("items", entries);
         jsonToReturn.put("status", "ok");
-        return jsonToReturn;
+        return getJSONCleaned(jsonToReturn);
     }
+
+    private JSONObject getJSONCleaned(JSONObject jsonToClean) {
```

Comment:
Suggest: reserve `get` prefix for JavaBean accessors. This method could be `cleanJson(JSONobject dirtyJSON)` or `filterJson(JSONobject input)` or `replaceProblematicCharacters(JSONObject withUnwashedCharacters)`